shared/studio/tabs/auth/loginUIPreview.tsx

```diff
@@ -1,8 +1,14 @@
+import {useEffect, useRef, useState} from "react";
```

Comment:
Didn't bother me before, but now I'm pretty bummed that we've implemented this entire UI in both React and vanilla JavaScript 😭 .